src/query/models/tags.go

```diff
@@ -23,9 +23,10 @@ package models
 import (
 	"bytes"
 	"fmt"
-	"hash/fnv"
 	"sort"
+	"strings"
 
+	"github.com/cespare/xxhash"
```

Comment:
nit: External import should appear below the internal ones separated by a new line.  (suprised linter is not catching these anymore)

Reply:
Yeah just noticed this one; I don't think our linter blocks builds any more, I've been seeing things that used to break it where it hasn't complained pretty often :/